homeassistant/components/canary/camera.py

```diff
@@ -105,6 +108,8 @@ def __init__(
             model=device.device_type["name"],
             name=device.name,
         )
+        self._image: bytes | None = None
+        self._expires_at = dt_util.utcnow() - FORCE_CAMERA_REFRESH_INTERVAL
```

Comment:
Is this 15 minute ago?

Reply:
I see, it's arbitrary since there is no image set.   The math here i think is not needed. Maybe just use `datetime.min` so it might be more clear its a placeholder small value?

Reply:
set to `dt_util.utcnow()` no math needed and works the same as before.